python/nav/web/info/room/views.py

```diff
@@ -48,6 +48,13 @@
     RACK_CENTER: 'center',
     RACK_RIGHT: 'right',
 }
+# See section 3.2.4 of RFC 3635:
+ETHERNET_INTERFACE_TYPES = (
+    6,  # ethernetCsmacd
+    62,  # fastEther
+    69,  # fastEtherFX
+    117,  # gigabitEthernet
```

Comment:
My lack of netops experience shows here, but: Prior to this commit, I assume the only valid value was 6? And you filtered on whether iftype was set or not in a roundabout way?

I'm assuming the non-6 entries are what you have found in the wild? If you find others, will you add them to this list, or would it be better to check if iftype is set at all (with `exclude(iftype__isnull=True)` maybe). A comment as to what to do about future iftypes not in the list would also be handy right here, or *why* we check for these specific ones and not others.

Reply:
I might suggest moving the list of ethernet values to a member of the `manage.Interface` model instead. What do you think, @hmpf ?

Reply:
That and some comments, because as is the code is still confusing, with magic numbers and so on. Nowhere does it say that "there are other magic numbers in use, but we only care about this tiny subset". Not Monday morning/Friday afternoon/stress-compatible code :) "Explicit is better than implicit." and "In the face of ambiguity, refuse the temptation to guess." in this case I think.

Reply:
.. on the gripping hand, maybe we should just merge this and make a new issue: "Make ETHERNET_INTERFACE_TYPES future caffeine deprived programmer proof", to save time.

Reply:
I've moved the definitions around and provided some more exposition in the comments, so I'm going to consider this resolved now (even though the entire PR was re-made in #2314